packages/typegpu/src/data/numeric.ts

```diff
@@ -1,28 +1,105 @@
 import * as TB from 'typed-binary';
+import { RecursiveDataTypeError } from '../errors';
 import type { TgpuData } from '../types';
+import alignIO from './alignIO';
 import { SimpleTgpuData } from './std140';
 
+const primitiveNumeric = (
+  schema: TB.Uint32Schema | TB.Float32Schema | TB.Int32Schema,
+  code: string,
+) => {
+  return {
+    __unwrapped: undefined as unknown as number,
```

Comment:
Add a comment like in the other schemas, that this is just a type token.
```suggestion
    __unwrapped: undefined as unknown as number,
```